src/utils.rs

```diff
@@ -0,0 +1,19 @@
+#[cfg(feature = "std")]
+pub fn unsafe_alloc_buffer(len: usize) -> Vec<u8> {
```

Comment:
Since above we don't use `pub mod utils;` I guess this can't be imported from outside the create.